docs/forms/FORM_ERRORS.md

```diff
@@ -1,38 +1,19 @@
-[Form errors](#introduction)
-
-- [Form errors](#form-errors)
-    - [Form error store](#form-error-store)
-      - [Error object model](#error-object-model)
-    - [Creating error object](#creating-error-object)
-      - [Custom errors](#custom-errors)
-      - [Basic/primitive errors](#basicprimitive-errors)
-      - [Build error messages](#build-error-messages)
-    - [Display inline error messages](#display-inline-error-messages)
-      - [Basic errors](#basic-errors)
-      - [Custom errors](#custom-errors-1)
-    - [Add errors to the store](#add-errors-to-the-store)
-    - [Errors translation definition](#errors-translation-definition)
-      - [Custom errors](#custom-errors-2)
-      - [Primitive errors](#primitive-errors)
-
-<a name="introduction"></a>
-
 # Form errors
 
-Handling the form errors in Otter context (container/presenter, localization ...), it's a bit different from creating a form in a component and do all the logic there.
-<a name="store"></a>
+Handling the form errors in the Otter context (container/presenter, localization, etc.) is a bit different from creating a form in a component and doing all the logic there.
 
 ### Form error store
 
-To have the possibility to display inline error messages in the form and also in error panels (on the top of the page, above submit button ...) the best match is to have a dedicated store for the form errors. In this way we can listen to the store state and display the errors anywhere in the page.
-The store is provided in __@o3r/forms__ package. See [Form Error Store](https://github.com/AmadeusITGroup/otter/blob/main/packages/@o3r/forms/src/stores/form-error-messages/form-error-messages.state.ts) for more details and state object model.
-<a name="errormodel"></a>
+To have the possibility of displaying inline error messages in the form and also in error panels (on the top of the page, above submit button, etc.),
+the best option is to have a dedicated NgRX store for the form errors. This way we can listen to the store state and display the errors anywhere in the page.
+The store is provided in the __@o3r/forms__ package. See [Form Error Store](https://github.com/AmadeusITGroup/otter/blob/main/packages/@o3r/forms/src/stores/form-error-messages/form-error-messages.state.ts)
+for more details and to view the state object model.
 
 #### Error object model
 
-The store model object is __FormError__. See below the form errors object models.
+The store model object extends the `FormError` interface (see its object model below).
 
-- The __FormError__ contains an identifier for each component which has a form inside, plus the errors associated to that form.
+- The `FormError` interface contains an identifier for each component that has a form inside, plus the errors associated to that form.
 
 ```typescript
```

Comment:
Should we keep a duplicate of the interface here or add a link to the file with the interface definition?
It will avoid misalignment